dev/acf-export.json

```diff
@@ -584,12 +584,12 @@
         "description": ""
     },
     {
-        "key": "group_5df8fb174faec",
-        "title": "Degree Content Fields",
+        "key": "group_58fa427b4efb7",
+        "title": "Degree Fields",
         "fields": [
             {
-                "key": "field_5dfa8794d5761",
-                "label": "Sidebar Content",
+                "key": "field_5e9610175a3c2",
+                "label": "Modern Layout",
```

Comment:
We should probably only show fields specific to the Modern layout when the "Degree Modern" template is in use.  I think this would require moving these fields into a separate field group.